subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/verification/DependencyVerificationIntegrityCheckIntegTest.groovy

```diff
@@ -16,19 +16,15 @@
 
 package org.gradle.integtests.resolve.verification
 
-
 import org.gradle.api.internal.artifacts.ivyservice.CacheLayout
 import org.gradle.integtests.fixtures.ToBeFixedForInstantExecution
-import org.gradle.integtests.fixtures.ToBeFixedForVfsRetention
 import org.gradle.integtests.fixtures.cache.CachingIntegrationFixture
 import org.gradle.test.fixtures.file.TestFile
-import org.gradle.util.TestPrecondition
 import spock.lang.Issue
 import spock.lang.Unroll
 
 import static org.gradle.util.Matchers.containsText
 
-@ToBeFixedForVfsRetention(because = "https://github.com/gradle/gradle/issues/13135", failsOnlyIf = TestPrecondition.WINDOWS)
```

Comment:
There are a few more ignored test on `master` now. There is also `DependenciesAttributesIntegrationTest` and `GroovyJavaLibraryInteractionIntegrationTest`. I wonder if `RichConsoleCompositeBuildGroupedTaskFunctionalTest` should work again as well.